src/main/java/com/diffblue/demo/ecommerce/CustomException.java

```diff
@@ -0,0 +1,15 @@
+package com.diffblue.demo.ecommerce;
+
+public class CustomException extends Exception {
```

Comment:
Not sure `Custom` is a good name. Usually exceptions types convey some high-level reason what has gone wrong.